lib/form.js

```diff
@@ -96,29 +121,41 @@ _.extend(Form.prototype, {
     locals: function (/*req, res*/) {
         return {};
     },
+    _checkEmpty: function (req, res, callback) {
```

Comment:
Could this lifecycle be named differently, as I'd like to override it in the wizard:
https://github.com/UKHomeOffice/passports-form-wizard/blob/4aa46df28547a5464938bda745ece8e9b7be1223/lib/controller/index.js#L29-L35